conftest.py

```diff
@@ -61,10 +65,12 @@ def pytest_configure(config):
     # Make sure that the answers dir exists. If not, try to make it
     if not os.path.isdir(answer_dir):
         os.mkdir(answer_dir)
+    if not os.path.isdir(array_dir):
+        os.mkdir(array_dir)
```

Comment:
The following pattern (EAFP) is more robust in case there's a race condition (running pytest in parallel)
```suggestion
    for directory in (answer_dir, array_dir):
        try:
            os.mkdir(directory)
        except FileExistsError:
            pass
```